### DISCUSSION TITLE
Certain dailies get stuck or don't work at all.

### DISCUSSION THREAD

**Discussion by helltoe** [KIvoy/nikke-assistant#11] [Q&A]
hello, I know that you recently stated that you haven't worked on this for a while, but I was wondering if you knew what I was doing wrong as certain dailies do not work at all. such as arena, advising and simulation room. but the assistant seems to work on collecting stuff outpost/shop items. is there anything I'm missing that I need? or is it because it only works in the Chinese locale? any help/answers would be much appreciated.

**Comment by KIvoy** [comment]
You are right as in if you aren't using the simplified chinese version of the game, some of the functions won't work because the matching assets are based on the simplified chinese version. You can check which assets have simplified chinese on them, and replace them with the corresponding English assets for it to work.

**Reply by helltoe** [reply]
Sorry how would I do that?

**Reply by KIvoy** [reply]
@helltoe if you go to images, you can see each individual image assets used for each of the functions. Simply create another mirror copy with the English version of the assets, and then replace the image directory. Just make sure you screenshot the corresponding assets with the same game resolution (which can be easily achieved when you open the assistant as it auto-sets the game resolution at start).

If you want to continue to benefit from other updates of the app and help out other players like you who plays the English version of the game,  you can submit a PR with those assets and we can make the app adapt to multiple locales.
